ProcessLib/GroundwaterFlowProcess.h

```diff
@@ -215,6 +215,8 @@ class GroundwaterFlowProcess : public Process
         _A.reset(_global_setup.createMatrix(_local_to_global_index_map->dofSize()));
         _x.reset(_global_setup.createVector(_local_to_global_index_map->dofSize()));
         _rhs.reset(_global_setup.createVector(_local_to_global_index_map->dofSize()));
+        _linearSolver.reset(new typename GlobalSetup::LinearSolver(*_A));
```

Comment:
This will work for elliptic problems, but for transient problems the `reset()` has to be done in any iteration.